imod_coupler/__main__.py

```diff
@@ -29,31 +30,31 @@ def main():
 
     parser.add_argument(
```

Comment:
I think, we want to get rid of command line options and env variables completely as soon as we have a config file